week2/homework/src/index.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const actions = require('./actions');
+
+const args = process.argv;
+
+args.splice(0, 2);
+
+const retrieve = argument => args.includes(argument);
```

Comment:
what happens if i don't pass any command? `node index.js`?
I should get the `help` menu. but i get nothing instead :)

Reply:
Hi @keepitterron  Thanks for your feedback, I fixed it.
if (retrieve('add')) actions.add(process.argv[1]);
else if (retrieve('remove')) actions.remove(process.argv[1]);
else if (retrieve('list')) actions.list(process.argv);
else if (retrieve('reset')) actions.reset(process.argv);
else if (retrieve('help')) actions.help(process.argv);
else actions.help(process.argv);